ui/docs/reference/platform/auth/roles.md

```diff
@@ -91,8 +92,7 @@ a scope-set containing `"assume:project-admin:zap"` would expand to include
 
 ### Stars in Parameters
 
-*WARNING*: be careful using `*` in scopes that will be expanded with a
-parameterization, as results may not be what you expect.
+<Warning>Be careful using `*` in scopes that will be expanded with a parameterization, as results may not be what you expect. </Warning>
```

Comment:
Same here. An extra space at the end of the sentence.

<img width="141" alt="Screen Shot 2019-09-04 at 5 36 04 PM" src="https://user-images.githubusercontent.com/3766511/64293403-7e911180-cf3a-11e9-9943-6ab21a774d74.png">

Reply:
I am hoping for no spacing issue in recent PR.